### ISSUE TITLE
Unable to reinstall avocode plugin for sketch

### ISSUE THREAD

**Issue by paohlaa** [avocode/avocode#1595]
IMPORTANT: Please try to describe how did it happen.

I downloaded avocode in order to export my designs from Sketch, the plugin was not showing up in Sketch so I troubleshooted with the instructions given and still nothing. 

-----
App v. `2.14.8`
Avocode UID: null
```
Error: Sketch is not installed.
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1882336
```

**Comment by jakubzitny** [comment]
Hi @paohlaa, thanks for reporting this. Can you please give us some info for troubleshooting this? What macOS version do you have?  Where is your Sketch app installed?

As a workaround for now, please download the plugin manually from [here](https://avocode-media.s3.amazonaws.com/download/sketch-plugin-new/45.2.0/sketch-plugin-new-mac-45.2.0.sketchplugin.zip).

**Comment by paohlaa** [comment]
Thanks for getting back to me, I finally have the plugin working thank you!

Here is additional info:
*macOS version:* 10.12.6
- My sketch appears to be installed in the download folder. While I was
troubleshooting the plugin issue I deleted both avacode and sketch from the
Application folder. I was unable to fully delete sketch and reinstalled a
"new" Sketch to see where the path would go. As of now, the Sketch
application is only present in Downloads, not in the Application folder it
was in before.

**Comment by jakubzitny** [comment]
Hi there, we've just released Avocode `2.16.5`. Sketch app detection and plugin installation should work much better in it. Please update and let us know if you have any more problems.
